packages/backend-modules/mail/MailchimpInterface.js

```diff
@@ -14,16 +14,24 @@ const MINIMUM_HTTP_RESPONSE_STATUS_ERROR = 400
 const MailchimpInterface = ({ logger }) => {
   checkEnv(['MAILCHIMP_API_KEY', 'MAILCHIMP_URL', 'MAILCHIMP_MAIN_LIST_ID'])
   return {
-    buildApiUrl(path) {
-      return `${MAILCHIMP_URL}/3.0/lists/${MAILCHIMP_MAIN_LIST_ID}${path}`
+    buildApiUrl(audienceId, path) {
```

Comment:
```suggestion
    buildApiUrl(path, audienceId = MAILCHIMP_MAIN_LIST_ID) {
```
then it's safer if the function is used somewhere random we don't know about…